components/board.loading/R/loading_server.R

```diff
@@ -248,28 +248,8 @@ LoadingBoard <- function(id,
     ## Get the pgx folder. If user folders are enabled, the user email
     ## is appended to the pgx dirname.
     getPGXDIR <- shiny::reactive({
```

Comment:
yep we need to replace getPGXDIR by auth$options$user_dir, but since its so common in the loading board, I decided to do the trick within the reactive.

this will require a large amount of refactoring, as we need to enter each .R file and change fn arguments, remove parentethesis from getPGXDIR(), as its not a reactive anymore, etc.. 

maybe a new issue for a future to-do

<img width="836" alt="image" src="https://github.com/bigomics/omicsplayground/assets/28757711/36b6784c-2afe-4018-bac2-1aee3868f61b">

Reply:
> Is there any way to get rid of this reactive entirely? It seems unnecessary and could lead to confusion.

new issue created. Thansk Nick!